packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -843,5 +843,8 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       });
     }
   }
-
-}
+  private _renderCustomLabel(item: IDropdownOption) {
```

Comment:
Minor -- I don't think Fabric has any style guidelines on this, but I'd move this method up with all the other render functions, around line 545